cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -561,10 +551,6 @@ func (ht *hcsTask) close(ctx context.Context) {
 //
 // This call is idempotent and safe to call multiple times.
 func (ht *hcsTask) closeHost(ctx context.Context) {
-	ctx, span := trace.StartSpan(ctx, "hcsTask::closeHost")
```

Comment:
Add a log statement here too?

Reply:
It also might be better to just log inside the `Once.Do` so we know the one time we actually do something